### DISCUSSION TITLE
Unable to set any test environment for the tests execution

### DISCUSSION THREAD

**Discussion by Mujtaba-git** [akshayp7/playwright-typescript-playwright-test#65] [Q&A] [answered]
How can I set the environment here? I have watched the YouTube video doing the exactly same but still stuck here

![CleanShot 2024-06-06 at 00 26 06@2x](https://github.com/akshayp7/playwright-typescript-playwright-test/assets/58841791/e1fb3539-ac5b-42d4-b40a-4fded0b0ce86)

**Comment by Mujtaba-git** [comment]
Youtube video command screenshot

![CleanShot 2024-06-06 at 00 33 02@2x](https://github.com/akshayp7/playwright-typescript-playwright-test/assets/58841791/12fde38a-9653-43ff-a9e5-272c921dbc97)

**Comment by akshayp7** [comment, marked as answer]
Hi Thank you for supporting. Can you please follow below steps
https://github.com/akshayp7/playwright-typescript-playwright-test/issues/57#issuecomment-1999540284

**Comment by pramothvm** [comment]
@akshayp7 Hi Brother,

I am trying to use this project "playwright-java-cucumber" of yours in this repo. Could you please guide me how to execute the featurefile scenarios in Parallel execution. As currently it is executing in serial only. Your help is highly appreciated😊

Thanks and Regards,
Pramoth

**Reply by akshayp7** [reply]
Hi than you for your patience 
For executing test cases in parallel, provide a suitable tag @SmokeTest at the start of your test case name, then in package.json against test:parallel give the tag value and execute :
npm run test:parallel --ENV="qa"
